src/visualizr/anitalker/experiment.py

```
@@ -1,339 +1,343 @@
+            sampler,
+            start,
+            motion_direction_start,
+            audio_driven,
+            face_location,
+            face_scale,
+            ypr_info,
+            noisy_t,
+            control_flag,
+        )
+
+    def forward(self, noise=None, x_start=None, ema_model: bool = False):
```

Ruff (0.12.2)

Missing docstring in public method

(D102)